.github/workflows/contracts-deploy.yml

```diff
@@ -72,24 +72,39 @@ jobs:
     needs: ['build-deploy']
     steps:
       - uses: actions/checkout@v2
-        with:
-          token: ${{ secrets.ARABOT_PAT }}
       - uses: actions/download-artifact@v3
         with:
           path: artefacts
+      - name: Install dependencies
+        run: yarn install --pure-lockfile
+      - name: Compile
+        run: cd packages/contracts && yarn run build
       - name: Update Releases.md
         run: node .github/helpers/contracts/updateReleasesMD.js
       - name: Update active_contracts.json
         run: node .github/helpers/contracts/updateActiveContractsJSON.js
       - name: Update Subgraph manifests
         run: node .github/helpers/subgraph/updateManifest.js
+      - name: create Dummy DAOs erc20
```

Comment:
move these scripts after the commit. we wanna make sure to have the updated data in the repo and than try to create these daos